python/classify.py

```diff
@@ -104,15 +104,20 @@ def main(argv):
 
     if args.gpu:
         print('GPU mode')
+    else:
+        print 'CPU mode'
```

Comment:
Print statements without the parentheses break the Python 3 compatibility that @philkr added in #1923.

As @longjon said, 
> Regarding future PRs, yes, let's try to maintain Python 3 compatibility as long as it's easy to do so.